jax/_src/util.py

```diff
@@ -493,6 +492,18 @@ def maybe_named_axis(axis, if_pos, if_named):
     named = True
   return if_named(axis) if named else if_pos(pos)
 
+def get_logger() -> logging.Logger:
+  """
+  Instantiate a global logger named "jax" as a direct descendant of the root logger.
+  JAX-internal code only logs messages on this logger, by default at DEBUG priority.
+  """
+
+  jax_logger = logging.getLogger("jax")
+  jax_logger.setLevel(logging.DEBUG)
```

Comment:
Remove the code to set the logger level.

There's no need to override it. If we do nothing, then logs get propagated to the root logger (i.e., NOTSET, see the docs). The root logger by default prints to stderr everything at priority WARNING or above, and that's exactly the behavior we want. We don't want to see debug logs by default.